src/services/locale.service.ts

```diff
@@ -5,6 +5,15 @@
 import { Injectable } from '@angular/core';
 import { Subject } from 'rxjs';
 import { isDefined } from '../util';
+import { VIPServiceConstants } from '../constants';
+
+export interface VLocale {
+    readonly languageCode: string;
```

Comment:
Only the languageCode is required, right?

Reply:
Not exactly, all the fields are required.